### PR TITLE
Implement a way to load experimental english strings from _locales/_e…

### LINKED ISSUE #517: Implement a way to have "experimental" language strings

### PR DESCRIPTION
**Pull request by Jikstra**
…xperimental-en.json. Refactored localize.js a bit. Fixes #517

### CODE REVIEW

File: src/localize.js [hunk 1]
```diff
@@ -48,46 +48,55 @@ function translate (messages) {
...
+
+  if (!localeMessages && locale.indexOf('-') !== -1) {
+    // We couldn't load the file for the locale but it's a dialect. Try to fall
+    // back to the main language (example: de-CH -> de)
     locale = locale.split('-')[0]
-    file = localeFile(locale)
+    localeFile = retrieveLocaleFile(locale)
+    localeMessages = getLocaleMessages(localeFile)
+  } else if (!localeMessages) {
+    log.error(`Could not load messages for ${locale}`, locale)
```
Comment (ralphtheninja) [inline src/localize.js]: Remove last `locale`, redundant information.

File: src/localize.js [hunk 2]
```diff
@@ -48,46 +48,55 @@ function translate (messages) {
...
+  let experimentalMessages = getLocaleMessages(experimentalFile)
+  if (experimentalMessages) {
+    messages = merge(messages, experimentalMessages)
   } else {
-    locale = 'en'
-    messages = english
+    log.debug(`No experimental language file (${experimentalFile}) found`)
   }
 
+  console.log(messages['no_chat_selected_suggestion_desktop'])
```
Comment (ralphtheninja) [inline src/localize.js]: Remove please.

**Comment by ralphtheninja** [comment]
Gah, but the changes were made in the other PR. Closing again :)

**Comment by Simon-Laux** [comment]
Can you add a guide linked in or in the readme that shows on how to use it?

**Pull request closed without merging**